### ISSUE TITLE
compatibilty enhancement with Easy Magic?

### ISSUE THREAD

**Issue by Traehgniw** [Pedro270707/chiseled-enchanting#1]
Either I'm really, really unlucky or this doesn't seem to be compatible with the mod Easy Magic - the enchantment table with this and Easy Magic installed seems to be acting as though only Easy Magic was installed.

**Comment by PvtFudgepants** [comment]
Yeah this seems to be the case. I just sat there for like 15 minutes testing it to see if it works for my modpack and it didn't work a single time. Removed easy magic and tried it and it worked.

**Comment by NolanHewitt** [comment]
I would like to see this as well.

**Comment by Keaton42** [comment]
I would love this!!
